src/sentry/api/serializers/models/userreport.py

```diff
@@ -59,10 +59,18 @@ def serialize(self, obj, attrs, user):
 
 
 class ProjectUserReportSerializer(UserReportSerializer):
+    def __init__(self, environment_id_func=None):
+        self.environment_id_func = environment_id_func if environment_id_func is not None else lambda: None
```

Comment:
This should be `None` and allow the `GroupSerializer` constructor to manage the default value.